Patna/Readme.md

```diff
@@ -0,0 +1,35 @@
+Chandigarh Municipal Spatial Data
```

Comment:
Why does this Patna Folder mention and link data for Chandigarh?

Reply:
Sorry it came by mistake i am still working on it.